identity-account/src/account/account.rs

```diff
@@ -55,37 +46,15 @@ pub struct Account {
   storage: Arc<dyn Storage>,
   client_map: Arc<ClientMap>,
   actions: AtomicUsize,
-  did: IotaDID,
+  chain_state: ChainState,
+  state: IdentityState,
```

Comment:
I forgot to add this logic back into `process_update`, but technically we still have the `Batch` save in `Account::save`, so I'll re-add it. This will be decided with #432 anyway. Thanks!